ci-operator/config/openshift/troubleshooting-panel-console-plugin/openshift-troubleshooting-panel-console-plugin-release-0.2.yaml

```diff
@@ -20,7 +20,6 @@ images:
 promotion:
   to:
   - excluded_images:
-    - troubleshooting-panel-console-plugin
```

Comment:
This looks like it should have been the test image I think

Reply:
Yeah, I think the same.